ajax/libs/vue-paginate/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "vue-paginate",
+  "filename": "vue-paginate.min.js",
+  "version": "3.4.0",
+  "description": "A simple vue.js plugin to paginate data",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/TahaSh/vue-paginate.git"
+  },
+  "keywords": [
+    "paginate",
+    "pagination",
+    "page",
+    "chunk",
+    "navigate",
+    "slice",
+    "vue"
+  ],
+  "author": "Taha Shashtari",
+  "license": "MIT",
+  "npmName": "vue-paginate",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.gz)"
```

Comment:
@extend1994 I think you should discuss first before take any action especially in this case looks like two of your mates have the same thought with you.